mw-webapp/src/component/modal/ModalContent/ModalContent.module.scss

```diff
@@ -24,8 +23,7 @@ $modal-animation-time: 150ms;
   overflow: auto;
   max-width: 95vw;
   max-height: 90vh;
-  padding: 20px 45px;
-  border: 2px solid var(--generalPrimaryColor);
+  padding: $paddingExtraBig;
   border-radius: $secondaryBorderRadius;
   animation: contentShow $modal-animation-time cubic-bezier(0.16, 1, 0.3, 1);
   background-color: var(--primaryBgColor);
```

Comment:
please check lines 49, 50. use token for border-radius

Reply:
fixed